Gemfile.lock

```diff
@@ -598,7 +598,8 @@ GEM
     zeitwerk (2.6.16)
 
 PLATFORMS
-  ruby
+  arm64-darwin-23
```

Comment:
We should probably keep `ruby` as well

Reply:
This line was removed when I ran `bundle _2.5.14_ install` 🤔

Reply:
Yeah, It's probably due to your system architecture. We have to make sure it works for all of them, so I say let's add it back manually

Reply:
It's the same problem with `nokogiri`. I see now that it's installing `nokogiri (1.16.6-arm64-darwin)`. But I don't know how to fix it cause if I add the `ruby` line manually when I run `bundle install,` it removes the line again 🥲

Reply:
Yeah but we still need to fix this. I tried building the Dockerfile locally and got a error due to a missing platform
<img width="1425" alt="image" src="https://github.com/rootstrap/rails_api_base/assets/26656167/4f7db157-5ab5-42f9-8036-51dc7496e88a">

Running `bundle lock --add-platform ruby` adds back the ruby platform and the Docker build starts to work again. I also manually removed the other platforms and only left `ruby` and it works on my local env as well. I think that's probably the best workaround